packages/docs/docusaurus.config.js

```diff
@@ -10,7 +10,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/okaidia');
 const config = {
   title: 'DXOS',
   tagline: 'The Decentralized Operating System',
-  url: 'https://dxos.org',
+  url: 'https://qa.dxos.org/',
```

Comment:
can you make this dependent on an env var? We'll need this to change depending on which place it's being deployed